doc/articles/get-started-vscode.md

```diff
@@ -1,74 +1,118 @@
 ## VS Code
 
-This guide will walk you through the set-up process for building WebAssembly apps with Uno, under Windows, Linux or macOS.
+This guide will walk you through the set-up process for building WebAssembly apps with Uno under Windows, Linux, or macOS.
 
 ### Prerequisites
+
 * [**Visual Studio Code**](https://code.visualstudio.com/)
 
+* [**Mono**](https://www.mono-project.com/download/stable/)
+
+* [**.Net Core SDK**](https://dotnet.microsoft.com/download) (**version 3.1.8** or later)
+
+    > Use `dotnet --version` from the terminal to get the version installed.
+
 ### Create an Uno Platform project
 
-1. Launch Code, then in the terminal type the following to install the Uno Platform templates:
+#### Install Uno Platform Template
+
+Launch Visual Studio Code and open a new terminal:
+
+In the terminal type the following to install the Uno Platform templates:
+
 ```bash
 dotnet new -i Uno.ProjectTemplates.Dotnet
 ```
-2. In the terminal type the following to create a new project:
+
+#### Create the project
+
+In the terminal type the following to create a new project:
+
 ```bash
 dotnet new unoapp -o MyApp -ios=false -android=false -macos=false -uwp=false --vscodeWasm
 ```
 
+> `MyApp` is the name you want to give it to your project.
```

Comment:
```diff
- is the name you want to give it to your project.
+ is the name you want to give to your project.
```
@davidjohnoliver not sure if I'm not mistaking here ;) thanks in advance for the feedback !

Reply:
yep @agneszitte-nventive you're correct :)